users.js

```diff
@@ -883,9 +872,9 @@ class User {
 			} else if (userType === '4') {
 				this.autoconfirmed = userid;
 			} else if (userType === '5') {
-				this.lock(false, userid + '#permalock');
+				Punishments.lock(userid, false, userid + '#permalock');
 			} else if (userType === '6') {
-				this.ban(false, userid);
+				Punishments.ban(userid, false);
```

Comment:
the user is the name used by permalocks etc? That one's not mandatory so I'd rather have that in the back.